### DISCUSSION TITLE
Request to Add Custom Fields for User Registration on Sunbird ED

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Lern/Community#182] [Query]
I hope this message finds you well. We are reaching out to request assistance in **adding two custom fields to the user registration process on our Sunbird ED portal.**

**Details of the custom fields required are as follows**

1. Bio (Descriptive Field): This field will allow users to provide additional information about themselves during the registration process. It should be a free-text field where users can enter a description of their background, interests, or any other relevant information.

2.Designation (Dropdown): This field will be a dropdown menu where users can select their designation from a predefined list of options. This dropdown menu should include common designations such as Student, Teacher, Administrator, etc.

Additionally, we need to ensure that the values entered in these custom fields are saved with the respective user profiles upon registration. Furthermore, we should have the ability to read and update these custom field values as needed.
We kindly request your guidance on how we can achieve this scenario within the Sunbird ED platform.

**we would appreciate instructions on how to:**
**Add the custom fields to the user registration form**.
**Ensure that the values entered in these fields are saved with the user profiles.**
**Provide functionality to read and update the custom field values as required.**

Any documentation, tutorials, or support you can provide to assist us in implementing these custom fields would be greatly appreciated.
Thank you for your attention to this matter. We look forward to your guidance and support in achieving this customisation on our Sunbird ED portal.

 @reshmi-nair, @rhwarrier, @komalm, @manishakapadnis

**Comment by mohitga** [comment]
@AmiableAnil  @PoojaJManjooran  
Please look into this request

**Comment by AmiableAnil** [comment]
@rajeevsathish Please look into this.

**Comment by ManojNathIC** [comment]
Hi @rhwarrier @AmiableAnil @PoojaJManjooran @rajeevsathish

We have decided to use the PostgreSQL external database to store designation and bio.

Please let us know if this approach is good, or Any documentation, tutorials, or support you can provide to help us implement these custom fields on our Sunbird ED portal.

cc:  @komalm, @manishakapadnis @paritshivani
